src/libponyc/type/cap.c

```diff
@@ -86,29 +70,25 @@ bool is_cap_sub_cap(token_id sub, token_id subalias, token_id super,
       return false;
   }
 
-  if((sub == super) || (super == TK_TAG) || (super == TK_CAP_ANY_BIND))
+  if((sub == super) || (super == TK_TAG))
     return true;
 
-  // Every possible value of sub must be a subtype of every possible value of
-  // super.
+  // Every possible instantiation of sub must be a subtype of every possible
+  // instantiation of super.
   switch(sub)
   {
     case TK_ISO:
       switch(super)
       {
-        case TK_ISO:
```

Comment:
This case was already handled by the `if (sub == super) return true` check above.